benchmarking/evaluations/metrics/envy_free_up_to_one_item.py

```diff
@@ -0,0 +1,66 @@
+import copy
+
+from api.models.team import Team
+from api.models.team_set import TeamSet
+from benchmarking.evaluations.interfaces import TeamSetMetric
+
+
+class EnvyFreenessUpToOneItem(TeamSetMetric):
```

Comment:
I feel like even after reading this name and the comment, I still don't understand what this is calculating conceptually

I know this is something from a paper, but it's good to keep in mind that the code should not be dictated by the specific naming conventions and terminology of papers -- and if it actually is a non-generalized metric that only applies for the specific algorithms from other papers, this will devolve into the kind of function destined for "never use this again once this course ends"